include/deal.II/fe/fe_tools.h

```diff
@@ -1211,9 +1213,8 @@ namespace FETools
    * If no finite element can be reconstructed from this string, an exception
    * of type @p FETools::ExcInvalidFEName is thrown.
    *
-   * The function returns a pointer to a newly create finite element. It is in
-   * the caller's responsibility to destroy the object pointed to at an
-   * appropriate later time.
+   * The function returns a std::unique_ptr to a newly created finite element
+   * meaning the caller attains ownership over the returned object.
```

Comment:
picky, but 'obtain' is more correct here.

Reply:
Thanks!